wpilibj/src/main/java/edu/wpi/first/wpilibj/SpeedControllerGroup.java

```diff
@@ -77,7 +77,7 @@ public void stopMotor() {
   @Override
   public void pidWrite(double output) {
     for (SpeedController speedController : m_speedControllers) {
```

Comment:
This should call `set(output)` instead to match what PWMSpeedController does. `set()` has the same implementation as what is currently done here. Don't forget to change SpeedControllerGroup.cpp too.